includes/cli/class-setup.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Newspack setup wizard. Required plugins, introduction, and data collection.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\CLI;
+
+use WP_CLI;
+use WP_REST_Request;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Setup Newspack CLI command.
+ */
+class Setup {
+
+	/**
+	 * Sets up a testing site
+	 *
+	 * This is a command used to set up a testing site from scratch, installing required plugins and running the onboard wizard automatically
```

Comment:
I don't think we should treat it as "testing site" setup. Rather a "site setup", simply. The starter content could be optional in the command, but anyway it can be deleted later.

Reply:
I think it can evolve into a multi purpose command. I just wanted to explicitly say that to keep the scope small for now, and allow the script to make certain choices